desktop/components/commercial_filter/models/params.coffee

```diff
@@ -57,7 +60,14 @@ module.exports = class Params extends Backbone.Model
     _.uniq(@fullyQualifiedLocations.concat((@get('aggregation_partner_cities') || [])).concat @get('partner_cities'))
 
   whitelisted: ->
```

Comment:
Basically, https://github.com/artsy/force/blob/9549f11f9b2d49611c34e33382765f66644b90ab/desktop/apps/collect/client.coffee#L28-L29 is where we parse the URL to set the initial filter state.

Then, we fetch artworks and immediately trigger a 'change' in order to render things, which also causes the URL/history to start being managed client-side.

This file is where we update the URL, based on the current filter state (as well as removing a redundant `?page=1`, and any other params that aren't explicitly whitelisted). So, we allow the whitelisting to include _any_ params that start with `utm`.

Reply:
Makes sense. I've been in both these classes today as i'm trying to circumvent some of the aggregations in the query to see if that speeds things up.

Reply:
Cool, lmk if you want any help or to pair.